modules/distribution/pom.xml

```diff
@@ -454,6 +454,36 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.0.0</version>
```

Comment:
shall we add this to pluginManagement in the root pom and remove the version from here?